docs/dataset.md

```diff
@@ -28,7 +29,7 @@ Please download the datasets from the official websites. It is recommended to sy
 
 #### 1.2 Multiple Object Tracking
 
-- For the training and testing of multi object tracking task, only one of the MOT Challenge datasets (e.g. MOT17) is needed.
+- For the training and testing of multi object tracking task, only one of the MOT Challenge datasets (e.g. MOT17) and CrowdHuman dataset are needed.
```

Comment:
Rephrase. 
1. Note it is the MOT Challenge dataset, we must use itself and can use CrowdHuman dataset as complementary.